evergreen/convert-client-cert-to-pkcs12.sh

```diff
@@ -25,6 +25,21 @@ openssl pkcs12 -export -in "${CLIENT_PEM}" \
   -name "Drivers Client Certificate" \
   -password "pass:${MONGO_X509_CLIENT_CERTIFICATE_PASSWORD}"
 
+if [[ "$OS" =~ MAC|Mac|mac ]]; then
+  # this function is not presented on mac OS
```

Comment:
Do you mean:

      # this function is not available on mac OS

I think where you said "presented" you mean "available"

Reply:
done